.gitignore

```diff
@@ -9,3 +9,4 @@ Module.symvers
 *.mod.c
 *.o
 modules.order
+dummy.c
```

Comment:
Wait, why does this move? We're still doing a `make -C kernel-cflags-finder`, no?

Reply:
I had a `dummy.c` somewhere I didn't expect it. It's possible it came from some random intermediate debugging state.